src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyTestSameAndLiteralDo.java

```diff
@@ -67,12 +67,12 @@ public void foo4() {
 
     public void foo5() {
         do {
-        } // violation
+        } // violation ''}' at column 17 should be on the same line with while'
```

Comment:
```
expected to match: 70:(?:\d+:)?\s.*'}' at column 17 should be on the same line with while.*
but was          : 70:9: '}' at column 9 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).
```

Reply:
![Screenshot_20220418-211410](https://user-images.githubusercontent.com/812984/163918981-1512aa55-ed74-4421-9cc9-3b620e0e24e2.png)

Regexp that you wrote is not matching to message, please update your regexp in input file